_posts/2023-05-23-GUIs-in-PyQt5.md

```diff
@@ -0,0 +1,263 @@
+# Ramin Honary
+
+  - Worked at Cross Compass since 2018
+  - Platform Team development, full stack
+  - Worked primarily on Manufacia
+  - Currently on working on ML-Platform
+
+## Why I use PyQt
```

Comment:
1. Remove "will demo", this blog is for people outside and they do not watch the movie.

1. Only the name "Pattern matching kit", "Image cropping kit" will not show what it is. Better to describe the background why these tools are developed. Then about the features requested.
You can use the term "AI Factory" service for which we offer customers to report the results of AI model we create from the image datasets provided by customers.
Consultants without coding ability sometimes have to properly handle images provided by customers before AI creation.

1. Better to mention several options here with advantage and disadvantage for the purpose of the tool to select PyQt objectly from your technical point of view. Your subjective opinion is fine, too.